widget/controllers/widget.item.controller.js

```diff
@@ -74,6 +75,20 @@
           }
         };
 
+        WidgetItem.addKeyboardView = function () {
```

Comment:
I don't think this is a good idea to use isIOS variable. Logic is not ok at all. For example, if I come to your code later on and see this variable, I will assume that you always say if this device is iOS or not. But in your case, you are binding keyboard logic to this field if you understand me. You should only put: 

$scope.isIOS  = /iPad|iPhone|iPod/.test(navigator.userAgent); 

And leave it like that. Don't change the values based on "keyboard view" function.

Reply:
Yeah I agree, I'll fix this issue